.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ concurrency:
 jobs:
   macos:
     name: macOS
-    runs-on: macos-14
+    runs-on: macos-latest
```

Comment:
I think this resolves to `macos-15`, which may be fine to explore using, but we've found that the newer runners are usually less available when first introduced. Is it possible to stick with `macos-14` for now or is `macos-15` required for this PR? (If required, can we update this to be explicit so that CI doesn't automatically upgrade in the future?)

Reply:
Totally fine, `macos-14` it shall remain.